docs/installation.md

```diff
@@ -58,6 +58,15 @@ Or install a specific version with:
 ./install-publisher.bash 1.0.beta1
 ```
 
+> [!WARNING]
+> Linux users need to have `inotify-tools` installed for the VSCode filewatcher to work properly.
```

Comment:
Is there a link we could add to the vscode docs that explain this?

Also is this true for positron? Or does positron bundle/otherwise get around this?

Reply:
vscode talks about needing to update settings in inotify but i don't see anywhere they call it out as a dependency:

https://code.visualstudio.com/docs/setup/linux#_visual-studio-code-is-unable-to-watch-for-file-changes-in-this-large-workspace-error-enospc


unfortunately, I updated my mac today and Rosetta will not work with Linux AMD on Parallels (which is all Positron provides, no ARM installer) so I can't confirm.

Reply:
> Is there a link we could add to the vscode docs that explain this

Are you referring to our `docs/vscode.md` file or _the_ VS Code docs?

Reply:
>  _the_ VS Code docs?

This. It would be great if we showed that this isn't a special publisher issues, it's a general vscode thing.

Reply:
> > _the_ VS Code docs?
> 
> This. It would be great if we showed that this isn't a special publisher issues, it's a general vscode thing.

Everywhere I look I only see references about updating inotify to work with large directories.  It seems most Linux distributions come with `inotify` out of the box so maybe this is more of a VM issue.

Reply:
Sorry I missed this on Monday

> Everywhere I look I only see references about updating inotify to work with large directories. It seems most Linux distributions come with inotify out of the box so maybe this is more of a VM issue.

Yeah, that sounds right — I've had the (displeasure) of working on things that use inotify separately before and never had an issue personally with it not being generally available. 

I do still think it would be beneficial (to future-us if no one else) to link out to a canonical resource of some sort that this is a vscode thing, it's not because we're doing something special or bespoke